packages/shared/src/types/helpers.ts

```diff
@@ -0,0 +1,6 @@
+// would need to move all the types from api/src/database/helpers to type this one better
+export declare type Helpers = {
+	date: any;
+	st: any;
+	schema: any;
+};
```

Comment:
Here moving all the type to properly type `Helpers` seems too much of a refactoring, that's why I just make a new type with only the structure and not the type of the properties inside. 

The 'real' type is in the api, here https://github.com/directus/directus/blob/d7f27830e43015ef111c297172ab46ebca29c97b/api/src/database/helpers/index.ts#L25

This duplicate `type Helpers` is used in `RelationsService` https://github.com/directus/directus/blob/d7f27830e43015ef111c297172ab46ebca29c97b/packages/shared/src/services/relations.ts#L14)